core/types/tx_pool.go

```diff
@@ -11,8 +11,10 @@ import (
 )
 
 const (
-	//MaxPoolTransactionDataSize is a heuristic data limit for DOS prevention 1.25Mb
-	MaxPoolTransactionDataSize = 1280 * 1024
+	//MaxP2PNodeDataSize is a 1.25Mb heuristic data limit for DOS prevention on node message
+	MaxP2PNodeDataSize = 1280 * 1024
```

Comment:
Yes, I will have a announcement about the change. But currently because of spam txns are using 2 have gas price, lots of users/Dapps can't get their txn accepted anyway. It's better to make the move now to correct the gas price limit.

Reply:
that's the old txn size limit we used, it may not be optimal, but don't want to change that now.

Reply:
Oh, yea. It is the transaction limit 2 days ago. so it shall be fine.